pandas/core/series.py

```diff
@@ -958,7 +971,7 @@ def __unicode__(self):
         max_rows = (height if get_option("display.max_rows") == 0 else
                     get_option("display.max_rows"))
 
-        self.to_string(buf=buf, name=self.name, dtype=self.dtype,
+        self.to_string(buf=buf, name=self._name, dtype=self.dtype,
```

Comment:
you will *always* use ``.name``, never ``._name``, except in the actual properties (or in the pickle routines when you need to manipulate it).